recipe/meta.yaml

```diff
@@ -42,6 +42,7 @@ test:
     - pip check
     - pytest hvplot
   requires:
+    - python 3.10
```

Comment:
The correct way, if the min is 3.10, is to use `{% set python_min = "3.10" %}` at the top and re-render.

Reply:
One issue I had was that even if `bokeh_sampledata` is declared as `bokeh_sampledata # [py>=310]` in the test dependencies, it was attempted to be installed with Python 3.9.

I was in a rush, so went for a solution that worked. Our next release 0.12.0 will likely drop Python 3.9 support, I'll fix all that when we get there :).

Thanks for your support!